Components/PlayerList/index.js

```diff
@@ -0,0 +1,89 @@
+import useSWR from "swr";
+import styled from "styled-components";
+import { useState } from "react";
+
+const StyledLetterList = styled.ul`
```

Comment:
Since you do not put `<li>` elements inside this should not be an `ul`. You're already setting `role="list"` so you're good to just do: 

```suggestion
const StyledLetterList = styled.div`
```

Reply:
Alternatively you could wrap all Buttons in an `<li>` ;)